### DISCUSSION TITLE
Version Control with Strapi – sync admin panel changes among the team

### DISCUSSION THREAD

**Discussion by erikfenriz** [strapi/strapi#6948] [Legacy Topics]
Hello.
We are looking for a way to sync the changes we make in the admin panel among team members. 
Is there a way to do that so no changes get lost using VCS or maybe by putting the admin panel on a remote resource where it will serve as a single source of truth?

**Comment by derrickmehaffy** [comment]
Not sure I entirely understand your question?

**Reply by erikfenriz** [reply]
When we make changes in the admin panel, for instance, Roles and Permissions or manually changing a field in database, we want them (changes) to be synchronized (version control) among the team. We noticed that these changes are saved in .tmp/data.db but by default it is gitignored and we have concerns regarding data collisions. 
To rephrase the question: what's the best workflow with Strapi to sync local admin panel changes for a team?

**Reply by jorisw** [reply]
I have the same question, just chiming in to elaborate:

- We intend to use Strapi as an API generator for our data model
- We intend to iterate on this data model
- We'd like to know how iterations on the data model configuration, are best version controlled. 
- Ideally, a central configuration file would exist, in which the content types (and their relationships) are configured, rather than the entire API endpoint JS code tree being the canonical place where this is represented. The best example of such a single file would be the GraphQL export `schema.graphql` that appears to be generated by the graphql plugin.
- If such a central definition file already exists, I'd love to know about it
- This would allow us to develop our data model through code, document changes to it in Git, instead of managing changes through the admin panel.

**Reply by derrickmehaffy** [reply]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).

**Comment by x0ptr** [comment]
@erikfenriz you could use a Database like mysql/mongo etc. not sqlite.

@jorisw and you want an configuration file where you have roles etc. predefined and can be loaded while booting strapi? I guess this link maybe helps you why its not currently there:
https://forum.strapi.io/t/database-migration-deployment-questions/434/9
